internal/resources/local/writer.go

```diff
@@ -49,11 +49,11 @@ func (writer *FSWriter) Write(ctx context.Context, resources *resources.Resource
 		return nil
 	}
 
-	logger := logging.FromContext(ctx).With(slog.String("directory", writer.Directory))
+	logger := logging.FromContext(ctx).With(slog.String("path", writer.Path))
 	logger.Debug("Writing resources", slog.Int("resources", resources.Len()))
 
 	// Create the directory if it doesn't exist
-	if err := ensureDirectoryExists(writer.Directory); err != nil {
+	if err := ensureDirectoryExists(writer.Path); err != nil {
```

Comment:
Should we also add a guard for the case where `writer.Path` exists but isn't a directory?